src/platform/mock-local-home-platform.ts

```diff
@@ -40,8 +49,10 @@ export class MockLocalHomePlatform {
   }
 
   //  Singleton getter
-  public static getInstance(): MockLocalHomePlatform {
-    if (!MockLocalHomePlatform.instance) {
+  public static getInstance(
+    resetState: boolean = false
```

Comment:
Do you think we'll ever want to include more parameters at some point to this getter? Should we include some JSDoc for this method?

Reply:
I'm not sure I want to allow other parameters in `getInstance()` since that complicates the assumptions classes could make when obtaining the singleton instance through this getter.  The reset state parameter should only be used externally when setting up a test, and not internally in the `MockLocalHomePlatform` or `AppStub`, for example.  Will add a JSDoc.